examples/image-streamer/build_plan.rb

```diff
@@ -104,6 +126,9 @@
 puts "\n#Removing a build plan with id #{item2['uri']} and name #{item2['name']}:"
 item2.delete
 puts "\n#Build plan with id #{item2['uri']} and name #{item2['name']} removed successfully."
+puts "\n#Removing a build plan with id #{plan_script2['uri']} and name #{plan_script2['name']}:"
+plan_script2.delete
+puts "\n#Plan script with id #{plan_script2['uri']} and name #{plan_script2['name']} removed successfully."
 puts "\n#Removing a build plan with id #{item3['uri']} and name #{item3['name']}:"
```

Comment:
Is `plan_script2` a build plan or a plan script? See lines 129 and 131.

Reply:
its a plan script which we create in the beginning of this example

Reply:
Can you then change the puts on line 129.
`puts "\n#Removing a **build plan** with id #{plan_script2['uri']}`